sal/class.json-api-site-jetpack.php

```diff
@@ -104,10 +104,22 @@ function has_wordads() {
 		return Jetpack::is_module_active( 'wordads' );
 	}
 
+	/**
+	 * Creates a nonce to be used with iframed block editor requests to a Jetpack site.
+	 * Disabled on the Jetpack side.
```

Comment:
sorry to be an obstacle here, but _why_ is it disabled and is _this_ the Jetpack side?

maybe something like this…

```php
/*
 * Creates a nonce to allow an iframed block editor to make authenticated requests to a Jetpack site.
 *
 * Disabled here in Jetpack because the nonce is only needed when making calls from the dotcom domains,
 * not when making calls on the same origin as we would be doing from `wp-admin/post.php` and friends.
 */
```